src/components/PageWithSideMenu.tsx

```diff
@@ -75,34 +73,34 @@ interface LinkButtonProps extends ButtonProps, Pick<LinkProps, 'to'> {
 }
 
 export const LinkButton = ({ isSelected, ...rest }: LinkButtonProps) => (
-  <li>
-    <Button variant={isSelected ? 'contained' : 'outlined'} size="large" LinkComponent={Link} {...rest} />
-  </li>
+  <Button
+    sx={{ bgcolor: isSelected ? 'primary.main' : 'white', ml: '1rem', width: 'min-content' }}
+    variant={isSelected ? 'contained' : 'outlined'}
+    LinkComponent={Link}
+    {...rest}
+  />
 );
 
 interface LinkIconButtonProps extends LinkButtonProps {
   icon: ReactNode;
 }
 
 export const LinkIconButton = ({ sx = {}, icon, ...rest }: LinkIconButtonProps) => (
-  <LinkButton sx={{ minWidth: 0, width: 0, ...sx }} {...rest}>
+  <LinkButton sx={{ minWidth: 0, width: 0, bgcolor: 'white', mr: '1rem', ...sx }} {...rest}>
```

Comment:
hvorfor `bgcolor: 'white'`?

Reply:
For at den også skal ha samme bakgrunn som LinkButton?

Reply:
Aha, skal være hvite de knappene der ja.. Kan du se på å gjøre sammen fiksen på knappene som ligger på fremsiden/søk også?

Reply:
gjør forsåvidt ikke noe, men kan vurdere å bruke `common.white` eller `background.default` i stedetet for `white` også, for å hjelpe litt ekstra om vi en dag skal få inn dark mode etc